draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.spec.tsx

```diff
@@ -1,42 +1,105 @@
 import React from "react"
-import { render } from "@testing-library/react"
-import * as AppearanceAnim from "./AppearanceAnim"
-import { Tooltip } from "./index"
-import "@testing-library/jest-dom"
-jest.mock("./AppearanceAnim")
-const AnimationProvider = AppearanceAnim.AnimationProvider as jest.Mock
+import { render, screen, waitFor } from "@testing-library/react"
+import { Button, IconButton } from "@kaizen/button"
+import ArrowIcon from "@kaizen/component-library/icons/arrow-right.icon.svg"
 
+import { Tooltip } from "./index"
 describe("<Tooltip />", () => {
-  beforeEach(() => {
-    AnimationProvider.mockReturnValue(<div />)
-  })
-  describe("When no animationDuration prop is given", () => {
-    it("animationDuration prop is passed to AnimationProvider as undefined", () => {
+  describe("Accessible descriptions", () => {
+    it("will render a button that will have an accessible description", async () => {
       render(
-        <Tooltip text="Example">
-          <div />
+        <Tooltip
+          text="Tooltip popup description for Kaizen Button"
+          display="inline"
+          isInitiallyVisible
+          position="below"
+        >
+          <Button label="More info" />
         </Tooltip>
       )
-      expect(AnimationProvider).toHaveBeenCalledWith(
-        expect.objectContaining({
-          animationDuration: undefined,
-        }),
-        {}
+
+      await waitFor(() => {
+        expect(
+          screen.getByRole("button", { name: "More info" })
+        ).toHaveAccessibleDescription(
+          "Tooltip popup description for Kaizen Button"
+        )
+      })
+    })
+    it("will render a IconButton that will have an accessible description", async () => {
```

Comment:
Given you've already tested this functionality above, I think we could do without this one. I don't feel super strongly, but we might want to consider that too many tests may eventually slow down our build.

Reply:
That's true, I suppose we can safely assume that even if we were to change the where the Icon was rendered in the button, it should always be a child of the interactive element